cypress/e2e/tests/migration/applicationinventory/assessment/copy_assessment/copy_assessment.test.ts

```diff
@@ -65,21 +68,22 @@ describe(["@tier2"], "Copy assessment and review tests", () => {
             .parent(trTag)
             .within(() => {
                 cy.get(selectBox).should("be.disabled");
-                cy.wait(2000);
+                cy.wait(2 * SEC);
             });
+        click('[aria-label="Close"]');
```

Comment:
This selector is used at multiple places. I think we should declare in views and use wherever needed.

Reply:
Done as well